migrations/2017-10-06-234455_fix_local_dbs_unique_gh_login/up.sql

```diff
@@ -0,0 +1,7 @@
+-- This migration should have no effect on production.
+-- Its sole purpose is to fix the local database of anyone who ran 2017-09-26-200549 locally--
+-- that migration can't be run on production because lower(gh_login) isn't unique on production.
+
+CREATE INDEX IF NOT EXISTS index_users_gh_login ON users (gh_login);
```

Comment:
We still want this index to be on the lowercase name though, don't we? Should we have another migration that is along the lines of:

```sql
DROP INDEX index_users_gh_login;
CREATE INDEX lower_gh_login ON users (lower(gh_login));
```

Reply:
ah, good call!